src/Controller/WorkQueueController.php

```diff
@@ -528,4 +539,79 @@ public function consentHistories($id, $type, ParticipantSummaryService $particip
             'consentStatusDisplayText' => WorkQueue::$consentStatusDisplayText
         ]);
     }
+
+    /**
+     * @Route("/view/delete", name="workqueue_view_delete")
+     */
+    public function workQueueViewDeleteAction(Request $request)
+    {
+        $workQueueViewDeleteForm = $this->createForm(WorkQueueViewDeleteType::class);
+        $workQueueViewDeleteForm->handleRequest($request);
+        if ($workQueueViewDeleteForm->isSubmitted() && $workQueueViewDeleteForm->isValid()) {
+            $workQueueViewId = $workQueueViewDeleteForm['id']->getData();
+            $workQueueView = $this->em->getRepository(WorkqueueView::class)->find($workQueueViewId);
+            if ($workQueueView) {
+                $this->em->remove($workQueueView);
+                $this->em->flush();
+                $this->addFlash('success', 'WorkQueue View Deleted');
```

Comment:
This is a minor detail, but can we make all of the flash messages consistent?

1. It looks like we use `Work Queue` in the navigation and page titles, so let's change `WorkQueue` to `Work Queue`.
2. This message is in title case, but others are sentence case. I think sentence case makes more sense (i.e. `Work Queue view deleted`)